frameworks/cassandra/tests/config.py

```diff
@@ -1,17 +1,21 @@
 """ Utilties specific to Cassandra tests """
 
 import os
+import logging
+import traceback
 
 import sdk_hosts
 import sdk_jobs
 import sdk_plan
 import sdk_utils
 
+log = logging.getLogger(__name__)
 
 PACKAGE_NAME = 'cassandra'
 DEFAULT_TASK_COUNT = 3
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
 DEFAULT_CASSANDRA_TIMEOUT = 600
+SERVICE_NAME = os.environ.get('SERVICE_NAME') or PACKAGE_NAME
```

Comment:
Yeah until we the change in soak-cluster-configurations gets in, setting this to `SOAK_SERVICE_NAME` makes it more explicitly clear as to why we have two difference service names in the general `config.py` file